docs/contribute/development.rst

```diff
@@ -192,6 +192,31 @@ Launch tests
 
 Pictures of the problem and videos are generated in cypress/videos and cypress/screenshots
 
+Setup to run rando synchronization locally
```

Comment:
Maybe it can ben advised that it is only for Rando V2 ?

Reply:
Sure, done.